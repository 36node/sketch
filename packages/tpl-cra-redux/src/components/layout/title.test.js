import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";

import Title from "./title";

it("should renders correctly", () => {
  const tree = renderer.create(<Title>somthing</Title>).toJSON();
  expect(tree).toMatchSnapshot();
});
