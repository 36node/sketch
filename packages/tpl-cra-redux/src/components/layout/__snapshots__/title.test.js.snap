// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should renders correctly 1`] = `
.c0 {
  font-size: 1.5rem;
}

<h1
  className="c0"
>
  somthing
</h1>
`;
